extensions/wikia/RTE/RTE.class.php

```diff
@@ -1,6 +1,10 @@
 <?php
 
 class RTE {
+	
+	//init modes
```

Comment:
This comment does not really offer any extra information 😉 Let's make it more detailed or remove it altogether.